src/main/docker/Dockerfile

```diff
@@ -14,40 +14,52 @@
 # limitations under the License.
 #
 
+# Preliminary image that includes Elasticsearch and related files
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.3.2 AS builder
+
 #
-# Preliminary image that downloads and extracts Elasticsearch
+# The actual image definition
 #
-FROM cafapi/opensuse-jre8:1 AS builder
+FROM cafapi/opensuse-jre8:1
 
-RUN zypper -n install which hostname
-RUN zypper -n install unzip
+ENV ELASTIC_CONTAINER=true
 
-# Download and Extract Elasticsearch
-ADD https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.3.2.zip /opt/
+COPY --from=builder /opt/jdk-10.0.2/ /opt/jdk-10.0.2/
+ENV JAVA_HOME=/opt/jdk-10.0.2
 
-RUN unzip /opt/elasticsearch-6.3.2.zip -d /opt/
+# Copy java certs and set link to them
+COPY --from=builder /etc/pki/ca-trust /etc/pki/ca-trust
+RUN ln -sf /etc/pki/ca-trust/extracted/java/cacerts /opt/jdk-10.0.2/lib/security/cacerts
 
-# Install the analysis-icu plugin for Elasticsearch
-RUN /opt/elasticsearch-6.3.2/bin/elasticsearch-plugin install analysis-icu
+# Instal packages
+RUN zypper --non-interactive -n install which hostname netcat-openbsd unzip wget
+RUN zypper --non-interactive clean --all
 
-# Configure logging
-COPY logging.yml /opt/elasticsearch-6.3.2/config
+RUN mkdir -p /usr/share/elasticsearch
+# Create group, user and set access rights to the elasticsearch base dir
+RUN groupadd -g 1000 elasticsearch && \
+    useradd -u 1000 -g 1000 -G 0 -d /usr/share/elasticsearch elasticsearch && \
+    chmod 0775 /usr/share/elasticsearch && \
+    chgrp 0 /usr/share/elasticsearch
 
-#
-# The actual image definition
-#
-FROM cafapi/opensuse-jre8:1
+WORKDIR /usr/share/elasticsearch
 
-RUN zypper -n install which hostname
+COPY --from=builder --chown=1000:0 /usr/share/elasticsearch/ /usr/share/elasticsearch/
+RUN /usr/share/elasticsearch/bin/elasticsearch-plugin install analysis-icu
 
-# Startup Elasticsearch and expose ports
-EXPOSE 9200 9300
-CMD /opt/elasticsearch/scripts/start.sh
+# Configure logging
+COPY logging.yml /usr/share/elasticsearch/config
+
+ENV PATH=/usr/share/elasticsearch/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 
-COPY --from=builder /opt/elasticsearch-6.3.2/ /opt/elasticsearch/
+COPY --from=builder --chown=1000:0 /usr/local/bin/docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
+RUN chgrp 0 /usr/local/bin/docker-entrypoint.sh && \
+    chmod g=u /etc/passwd && \
+    chmod 0775 /usr/local/bin/docker-entrypoint.sh
 
-COPY scripts/ /opt/elasticsearch/scripts/
-RUN chmod +x /opt/elasticsearch/scripts/*
+EXPOSE 9200 9300
+ENTRYPOINT ["/usr/local/bin/docker-entrypoint.sh"]
```

Comment:
Does overriding the entrypoint set [here](https://github.com/CAFapi/opensuse-base-image/blob/v1.1.1/src/main/docker/Dockerfile#L34) mean that the startup scripts won't be run?  Can this be called from the startup script instead?

Reply:
You are right, it was not running startup.sh script.